Tests/Data/Elliptic/square_100x100_ComponentTransport/square_1e4_heterogeneity.prj

```diff
@@ -201,16 +216,16 @@
     <linear_solvers>
         <linear_solver>
             <name>general_linear_solver</name>
-            <lis>-i bicgstab -p ilut -tol 1e-8 -maxiter 20000</lis>
+            <lis>-i bicgstab -p ilut -tol 1e-16 -maxiter 20000</lis>
```

Comment:
Why did you choose such a strong tolerance criterion (1e-16)?

Reply:
I have no idea on the setting of linear solver... @endJunction